### PR TITLE
fix wrong description message about the -s option

### PR DESCRIPTION
**Pull request by namusyaka**

### CODE REVIEW

File: lib/sinatra/main.rb [hunk 1]
```diff
@@ -5,7 +5,9 @@ module Sinatra
     require 'optparse'
     parser = OptionParser.new { |op|
       op.on('-p port',   'set the port (default is 4567)')                { |val| ParamsConfig[:port] = Integer(val) }
-      op.on('-s server', 'specify rack server/handler (default is puma)') { |val| ParamsConfig[:server] = val }
+      op.on('-s server', "specify rack server/handler (Of Sinatra's default servers, one with a rack handler will be adopted)") {
```
Comment (jkowens) [inline lib/sinatra/main.rb:8]: Could we simplify this to "specify rack server/handler"? I think explaining the default is too difficult here. The list of servers could potentially be overridden right?

Comment (namusyaka) [reply lib/sinatra/main.rb:8]: Yeah i thought it would be better but considered the consistency for our UI. And the behaviour may be confusing a bit :(

Comment (namusyaka) [reply lib/sinatra/main.rb]: I was rethinking about it and I concluded that it should be mentioned in our README, not command line.

**Comment by jkowens** [comment]
Looks good 👍

**Pull request merged**